### PR TITLE
Draft 027

### PR DESCRIPTION
**Pull request by ESTHER0533**
@deletosh add draft 027

### CODE REVIEW

File: 027.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+
+## Brief #027:What the heck is LegalTech – An introduction to and overview of the technology disrupting the legal profession
```
Comment (deletosh) [inline 027.md:2]: @ESTHER0533 can you please update this to follow the proper title syntax (adding the content of this, with the, "and more"

File: 027.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+### Content Tips for Law Firm Bloggers,How Much Should I Spend on My Legal Marketing Budget?,The Clubhouse App: Does It Live Up To Hype? and more...
+
+## This Week's Favorite
+
+[What the heck is LegalTech – An introduction to and overview of the technology disrupting the legal profession](https://www.planetcompliance.com/what-the-heck-is-legaltech-an-introduction-to-and-overview-of-the-technology-disrupting-the-legal-profession/) 
+
+Legal Technology is not a new concept though as technology has played an important part in the jobs of lawyers both in-house and at law firms. Two aspects though have created a movement that aims to change every angle of an industry:
```
Comment (deletosh) [inline 027.md:10]: @ESTHER0533 this block seems incomplete, can you comfirm. I don't see the point of ending in a ":"

**Review by deletosh — CHANGES_REQUESTED** [review]
@ESTHER0533 please see my review and update according

File: 027.md (hunk 1 above)
Comment (ESTHER0533) [reply 027.md:2]: @deletosh i dont really understand your inference on this

Comment (deletosh) [reply 027.md:2]: @ESTHER0533 this is the title of the newsletter. It should have something like "Brief #027: What the heck is LegalTech, Abridged Title from Another Article, Maybe One More, and more."

Comment (ESTHER0533) [reply 027.md:2]: @deletosh Do you mean other sum titles, they are indicated in this article Sir,
image

**Comment by ESTHER0533** [comment]
@deletosh Draft 027 updated

File: 027.md (hunk 1 above)
Comment (deletosh) [reply 027.md:2]: @ESTHER0533 this section



Is what should have the updated subject line

Comment (ESTHER0533) [reply 027.md:2]: @deletosh Do you me the heading should also be among the Sub headings i.e , the extracts?

Comment (deletosh) [reply 027.md:2]: @ESTHER0533 yes.

Comment (ESTHER0533) [reply 027.md:2]: @deletosh Sir, I cant seem to find this markdown file anymore, did you delete?

**Review by ESTHER0533 — COMMENTED** [review]
@deletosh  changes made

**Pull request merged**